src/test/resources/chunk_uplevel/src/main/composite.dita

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE dita PUBLIC "-//OASIS//DTD DITA Composite//EN" "ditabase.dtd">
+<dita>
+  <topic id="chunkmefirst">
+<title>First topic in main dir!</title>
+<body>
+<p><keyword conref="../common/reuse.dita#reuse/kwd"/></p>
+</body>
+<concept id="chunkmetoo">
+<title>Second topic in main dir!</title>
+<conbody>
+<p>thing</p>
+</conbody>
+</concept>
+</topic>
+</dita>
+
+<!--
```

Comment:
Maybe remove this comment because the log shown here is not what the final modifications will output.

Reply:
Shouldn't have been in there either way, leftover from my debugging of the test file itself. Thanks...